src/Symfony/Component/Mime/Tests/ExternalTemplatedEmailTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mime\ExternalTemplatedEmail;
+
+class ExternalTemplatedEmailTest extends TestCase
+{
+    public function testTemplateId()
+    {
+        $e = new ExternalTemplatedEmail();
+        $e->templateId('template_id');
+        $this->assertEquals('template_id', $e->getTemplateId());
```

Comment:
assertSame where possible

Reply:
Thanks for your feedback. It's updated :)